src/js/pages/admin.js

```diff
@@ -1,5 +1,66 @@
 import { listenForHistoricalReset } from "../modules/historical-reset.js"
+import React from "react"
+import { createRoot } from "react-dom/client"
+import AdminPanel from "../components/AdminPanel"
+import { simpleGet } from "../modules/utils.js"
+;(function () {
+  window.addEventListener("DOMContentLoaded", () => {
+    listenForHistoricalReset()
 
-window.addEventListener("DOMContentLoaded", () => {
-  listenForHistoricalReset()
-})
+    // Only show the admin redis panel on the summit and base sites
+    const siteLocation = window.APP_DATA
```

Comment:
yikes, something else was also missing :sweat:, we need to unpack the siteLocation from window.APP_DATA.
```suggestion
    const { siteLocation } = window.APP_DATA
```

Reply:
Good spot!